sklearn/exceptions.py

```diff
@@ -30,7 +30,8 @@ class NotFittedError(ValueError, AttributeError):
     ... except NotFittedError as e:
     ...     print(repr(e))
     ...                        # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
-    NotFittedError('This LinearSVC instance is not fitted yet'...)
+    NotFittedError("This LinearSVC instance is not fitted yet. Call 'fit' with
```

Comment:
Not sure why you changed this

Reply:
I had to change this, otherwise tests would not pass (link to previously failed test [here](https://travis-ci.org/scikit-learn/scikit-learn/jobs/481309007#L2125)), the reason being that the previous manual check had its own error message which was not consistent with the default 'check_is_fitted' error message.

Reply:
Of course. Thanks for the response.